x-pack/test/search_sessions_integration/tests/apps/dashboard/async_search/save_search_session_relative_time.ts

```diff
@@ -61,6 +61,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.header.waitUntilLoadingHasFinished();
       await PageObjects.dashboard.waitForRenderComplete();
 
+      // saving dashboard to populate map buffer. See https://github.com/elastic/kibana/pull/91148 for more info
+      // This can be removed after a fix to https://github.com/elastic/kibana/issues/98180 is completed
+      await PageObjects.dashboard.switchToEditMode();
+      await PageObjects.dashboard.clickQuickSave();
```

Comment:
Added a dashboard save before saving and loading the session. This helps with the flakiness by ensuring that the `mapBuffer` is saved before the session gets saved and loaded. This will prevent the cache misses which were occasionally failing this test. For the reasoning, see the caveat mentioned in this PR https://github.com/elastic/kibana/pull/91148.

After a fix to https://github.com/elastic/kibana/issues/98180 is created, this change will no longer be required.